<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="acceptDrops">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowIcon">
   <iconset theme="task-due">
    <normaloff>.</normaloff>.</iconset>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:rgb(170, 170, 255);
background: qlineargradient(x1:0, y1:0, x2:1, y2:0, stop:0 #1F618D, stop:1 #f0f0f0);
</string>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Rounded</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <widget class="QLabel" name="maxProfitLabel_2">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>350</y>
      <width>211</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Rockwell</family>
      <pointsize>9</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(255, 255, 255)

</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>400</y>
      <width>211</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Rockwell</family>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(255, 255, 255)</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>300</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Yu Gothic UI Semibold</family>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(255, 255, 255)</string>
    </property>
    <property name="text">
     <string>    OUTPUT:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>450</y>
      <width>211</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Rockwell</family>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(255, 255, 255)</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="optimizeButton_2">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>260</y>
      <width>209</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Optimize</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="trainingBudgetInput_2">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>140</y>
      <width>209</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(255, 255, 255)</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="energy">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>190</y>
      <width>211</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(255, 255, 255)</string>
    </property>
   </widget>
   <widget class="QLabel" name="opt">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>350</y>
      <width>211</width>
      <height>161</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(255, 255, 255)</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="transportationBudgetInput_2">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>90</y>
      <width>209</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(255, 255, 255)</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="capacityInput_2">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>40</y>
      <width>209</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(255, 255, 255)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>90</y>
      <width>209</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>10</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(255, 255, 255)</string>
    </property>
    <property name="text">
     <string>Enter Transportation Budget:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>140</y>
      <width>209</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>10</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(255, 255, 255)</string>
    </property>
    <property name="text">
     <string>Enter Training Budget:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>40</y>
      <width>209</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>10</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(255, 255, 255)</string>
    </property>
    <property name="text">
     <string>Enter Capacity</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>190</y>
      <width>209</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>10</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(255, 255, 255)</string>
    </property>
    <property name="text">
     <string>Enter Energy Usage:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>490</y>
      <width>211</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Rockwell</family>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(255, 255, 255)</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
